src/views/ModelPlan/CollaborationArea/index.tsx

```diff
@@ -0,0 +1,281 @@
+  );
+
+  // Updates state if session value changes
+  useEffect(() => {
+    setStatusChecked(statusCheckedStorage);
+  }, [statusCheckedStorage]);
+
+  // Sets the modal open state based on session state and suggested phase
+  useEffect(() => {
+    if (suggestedPhase && !statusChecked) setStatusPhaseModalOpen(true);
+  }, [suggestedPhase, statusChecked]);
+
+  //   useEffect(() => {
+  //     if (discussionID) setIsDiscussionOpen(true);
+  //   }, [discussionID]);
+
+  const favoriteMutations = useFavoritePlan();
+
+  const handleUpdateFavorite = (
+    modelPlanID: string,
+    type: UpdateFavoriteProps
+  ) => {
+    favoriteMutations[type]({
+      variables: {
+        modelPlanID
+      }
+    }).then(() => refetch());
+  };
+
+  return (
+    <MainContent
+      className="collaboration-area"
+      data-testid="collaboration-area"
+    >
+      <GridContainer>
+        <Grid desktop={{ col: 12 }}>
+          <Breadcrumbs
+            items={[
+              BreadcrumbItemOptions.HOME,
+              BreadcrumbItemOptions.COLLABORATION_AREA
+            ]}
+          />
+        </Grid>
+
+        {error && (
+          <ErrorAlert
+            testId="formik-validation-errors"
+            classNames="margin-top-3"
+            heading={collaborationAreaT('errorHeading')}
+          >
+            <ErrorAlertMessage
+              errorKey="error-document"
+              message={collaborationAreaT('errorMessage')}
+            />
+          </ErrorAlert>
+        )}
+
+        {message && (
+          <Alert slim type="success">
+            {message}
+          </Alert>
+        )}
+
+        {!loading && statusMessage && (
+          <Alert slim type={statusMessage.status} closeAlert={setStatusMessage}>
+            {statusMessage.message}
+          </Alert>
+        )}
+
+        {/* Wait for model status query param to be removed */}
+        {!data && (
+          <div className="height-viewport">
+            <PageLoading />
+          </div>
+        )}
+
+        {data && (
+          <Grid>
+            <Grid row className="collaboration-area__header">
+              <Grid desktop={{ col: 9 }}>
+                <PageHeading className="margin-top-4 margin-bottom-0">
+                  {collaborationAreaT('heading')}
+                </PageHeading>
+                <p
+                  className="margin-top-1 margin-bottom-2 font-body-lg"
+                  data-testid="model-plan-name"
+                >
+                  {collaborationAreaT('modelPlan', {
+                    modelName
+                  })}
+                </p>
+              </Grid>
+
+              <Grid desktop={{ col: 3 }} className="margin-top-4">
+                <div className="display-flex flex-justify-end">
+                  <FavoriteIcon
+                    isFavorite={isFavorite}
+                    modelPlanID={modelID}
+                    updateFavorite={handleUpdateFavorite}
+                    isCollaborationArea
+                  />
```

Comment:
FavoriteIcon component is missing the hover states that Natasha specified here: https://www.figma.com/design/smHRxJ9HIdywFV052vAc3Q/MINT-System-Library?node-id=524-205&t=Pk7FuMomhL7XqLfa-0

Reply:
Nice catch thanks!